core/reporter-api/src/main/java/org/arquillian/reporter/api/model/report/AbstractReport.java

```diff
@@ -1,14 +1,14 @@
 package org.arquillian.reporter.api.model.report;
 
-import java.util.ArrayList;
-import java.util.List;
-
+import org.arquillian.reporter.api.builder.report.ReportBuilder;
 import org.arquillian.reporter.api.model.StringKey;
 import org.arquillian.reporter.api.model.UnknownStringKey;
 import org.arquillian.reporter.api.model.entry.Entry;
-import org.arquillian.reporter.api.builder.report.ReportBuilder;
 import org.arquillian.reporter.api.utils.Validate;
 
+import java.util.ArrayList;
+import java.util.List;
+
```

Comment:
Yeah, if we want to go deeper to IDE settings we can share xmls... but maybe it's not really needed